### PR TITLE
6 create session controller

### LINKED ISSUE #6: Create Session Controller
Implements session controller likes class diagram.

### PR DESCRIPTION
**Pull request by BdsOliveira**

### CODE REVIEW

File: src/Helpers/Session.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public function __construct()
+    {
+        if (!session_id()) {
+            session_start();
+        }
+    }
+
+    public static function get(string $key): string
+    {
+        return self::has($key) ? $_SESSION[$key] : 'null';
```
Comment (danieldsf) [inline src/Helpers/Session.php]: Why null as string?

Comment (danieldsf) [reply src/Helpers/Session.php]: THis can bring ambiguity later

**Review by danieldsf — CHANGES_REQUESTED** [review]
Need some more clarification on where returning null as a string would be beneficial.

File: src/Helpers/Session.php (hunk 1 above)
Comment (BdsOliveira) [reply src/Helpers/Session.php]: FIxed.

**Review by danieldsf — APPROVED** [review]

**Pull request merged**